tests/integration/400.command.dev.test.js

```diff
@@ -363,353 +362,4 @@ test('should handle multipart form data when redirecting', async (t) => {
-        },
-      })
-
-    await builder.buildAsync()
-
-    await withDevServer({ cwd: builder.directory }, async (server) => {
-      const response = await got(`${server.url}/foo`).text()
-      t.is(response, '<html><h1>not-foo')
-    })
-  })
-})
-
-test('should use existing file when rule contains file extension and force=false', async (t) => {
-  await withSiteBuilder('site-with-shadowing-file-extension-force-false', async (builder) => {
-    builder
-      .withContentFile({
-        path: 'foo.html',
-        content: '<html><h1>foo',
-      })
-      .withContentFile({
-        path: path.join('not-foo', 'index.html'),
-        content: '<html><h1>not-foo',
-      })
-      .withNetlifyToml({
-        config: {
-          redirects: [{ from: '/foo.html', to: '/not-foo', status: 200, force: false }],
-        },
-      })
-
-    await builder.buildAsync()
-
-    await withDevServer({ cwd: builder.directory }, async (server) => {
-      const response = await got(`${server.url}/foo.html`).text()
-      t.is(response, '<html><h1>foo')
-    })
-  })
-})
-
-test('should redirect when rule contains file extension and force=true', async (t) => {
-  await withSiteBuilder('site-with-shadowing-file-extension-force-true', async (builder) => {
-    builder
-      .withContentFile({
-        path: 'foo.html',
-        content: '<html><h1>foo',
-      })
-      .withContentFile({
-        path: path.join('not-foo', 'index.html'),
-        content: '<html><h1>not-foo',
-      })
-      .withNetlifyToml({
-        config: {
-          redirects: [{ from: '/foo.html', to: '/not-foo', status: 200, force: true }],
-        },
-      })
-
-    await builder.buildAsync()
-
-    await withDevServer({ cwd: builder.directory }, async (server) => {
-      const response = await got(`${server.url}/foo.html`).text()
-      t.is(response, '<html><h1>not-foo')
-    })
-  })
-})
-
-test('should redirect from sub directory to root directory', async (t) => {
-  await withSiteBuilder('site-with-shadowing-sub-to-root', async (builder) => {
-    builder
-      .withContentFile({
-        path: 'foo.html',
-        content: '<html><h1>foo',
-      })
-      .withContentFile({
-        path: path.join('not-foo', 'index.html'),
-        content: '<html><h1>not-foo',
-      })
-      .withNetlifyToml({
-        config: {
-          redirects: [{ from: '/not-foo', to: '/foo', status: 200, force: true }],
-        },
-      })
-
-    await builder.buildAsync()
-
-    await withDevServer({ cwd: builder.directory }, async (server) => {
-      const response1 = await got(`${server.url}/not-foo`).text()
-      const response2 = await got(`${server.url}/not-foo/`).text()
-
-      // TODO: check why this doesn't redirect
-      const response3 = await got(`${server.url}/not-foo/index.html`).text()
-
-      t.is(response1, '<html><h1>foo')
-      t.is(response2, '<html><h1>foo')
-      t.is(response3, '<html><h1>not-foo')
-    })
-  })
-})
```

Comment:
These tests are all duplicates and are actually also in `500.command.dev.test.js`